sarathi/til.py

```diff
@@ -385,6 +386,27 @@ def fix_urls():
                     sys.stderr.write(f"Failed getting title for `{url}`")
     write_til_json(til_json)
 
+def print_fortune(args=None):
+    import subprocess
+    import fortune
+    
+    load_dotenv()
+    
+    params = ['fortune']
+    if args != None:
+        for i in args:
+            params.append(i)
+            
+
+    FORTUNE = os.getenv(FORTUNE_DIR)
```

Comment:
Again, I am not certain *why* you need a directory for fortune. What are you trying to accomplish? You do not need an configuration variable for this feature.